suro-core/src/main/java/com/netflix/suro/routing/MessageRouter.java

```diff
@@ -83,6 +89,13 @@ public void process(SuroInput input, MessageContainer msg) throws Exception {
                     } else {
                         sink.writeTo(msg);
                     }
+
+                    DynamicCounter.increment(
```

Comment:
I woud suggest move this before line 83. and maybe "routed" or "written" is clearer than "processed"?